mm2src/coins/tendermint/tendermint_tx_history_v2.rs

```diff
@@ -594,15 +615,26 @@ where
             events.reverse();
```

Comment:
caught this code block while review this PR.. please can you update to use  .`rev()` to reverse before collecting instead of reversing after calling collect. like this

```rust
let mut events: Vec<&Event> = tx_events
                .iter()
                .filter(|event| ACCEPTED_EVENTS.contains(&event.kind.as_str()))
                .rev()
                .collect();
```

so we can delete `events.reverse();`